mnefun/_mnefun.py

```diff
@@ -610,18 +610,24 @@ def fetch_raw_files(p, subjects, run_indices):
         raw_dir = op.join(subj_dir, p.raw_dir)
         if not op.isdir(raw_dir):
             os.mkdir(raw_dir)
-        finder_stem = 'find %s ' % p.acq_dir
-        # build remote raw file finder
-        fnames = get_raw_fnames(p, subj, 'raw', True, False, run_indices[si])
+        fnames = get_raw_fnames(p, subj, 'raw', True, False,
+                                run_indices[si])
         assert len(fnames) > 0
-        finder = (finder_stem +
+        # build remote raw file finder
+        for remote_dir in ['/sinuhe_data01', '/data101', '/sinuhe/data01',
+                           '/sinuhe/data02','/sinuhe/data03']:
```

Comment:
Ok.
I am having trouble remembering how to modify values of `params` attributes.
Line 378 in `__init__` is `self.acq_ssh = self.acq_dir = None` which is overridden by users arg I am failing to see why adding:
```
if acq_dir is not None:
self.acq_dir = [loc+'/%s' % self.acq_dir for loc in ['/sinuhe_data01',
                                                                             '/data101',
                                                                             '/sinuhe/data01',
                                                                             '/sinuhe/data02',
                                                                             '/sinuhe/data03']]
```
on the next line fails to change the value for `acq_dir`?